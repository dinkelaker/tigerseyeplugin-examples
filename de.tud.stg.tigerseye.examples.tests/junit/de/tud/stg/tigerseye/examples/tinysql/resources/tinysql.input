	println "Registered people:"
	
	List<Map> result = SELECT "NAME","AGE" FROM "PERSON" 
	
	result .each{
		
		Map row ->
		println "Name: ${row.Name} Age:${row.Age}";
		
		}