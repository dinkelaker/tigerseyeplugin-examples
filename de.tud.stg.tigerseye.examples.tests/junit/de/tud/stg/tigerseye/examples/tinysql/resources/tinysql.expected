	println "Registered people:"
	
	List<Map> result = selectFrom(
[
"NAME",
"AGE"
] as String[],
[
"PERSON"
] as String[]) 
	
	result .each{
		
		Map row ->
		println "Name: ${row.Name} Age:${row.Age}";
		
		}